Formula/azure-storage-cpp.rb

```diff
@@ -0,0 +1,23 @@
+class AzureStorageCpp < Formula
+  desc "Microsoft Azure Storage Client Library for C++"
+  homepage "https://azure.github.io/azure-storage-cpp"
+  url "https://github.com/Azure/azure-storage-cpp/archive/v5.1.1.tar.gz"
+  sha256 "29b4b0c47784ae541b0ef1b3693acba54e0d06bac78d139105a379ab1fd20333"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "cpprestsdk"
+  depends_on "gettext"
+  depends_on "libxml2"
+  depends_on "openssl"
+  depends_on "ossp-uuid"
+
+  def install
+    system "cmake", "-DBUILD_SAMPLES=OFF", "-DBUILD_TESTS=OFF", "Microsoft.WindowsAzure.Storage", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "true"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
Try to perform an operation without credentials (or with invalid credentials) and check that it fails?

Reply:
added test. Please see revision 85651eff676faff34ac287ff95a3620e4204508c

Reply:
@fxcoudert 
if you are OK with the changes, it would be great if you'd accept the formula and merge it into the master branch.

Thanks for your help!